src/components/LineSeries.tsx

```diff
@@ -60,13 +60,34 @@ export default function LineSeries(props: LineSeriesProps) {
     otherProps.markerStyle?.fill,
   ]);
 
+  if (hidden) return null;
+
+  const lineProps = {
+    id,
+    data: props.data,
+    xAxis: props.xAxis || 'x',
+    yAxis: props.yAxis || 'y',
+    lineStyle,
+  };
+  const errorBarsProps = {
+    data: props.data,
+    xAxis: props.xAxis || 'x',
+    yAxis: props.yAxis || 'y',
+    hidden: !displayErrorBars,
+    style: props.errorBarsStyle,
+    capStyle: props.errorBarsCapStyle,
+    capSize: props.errorBarsCapSize,
+  };
+
   return (
     <g>
-      {props.hidden ? null : <LineSeriesRender {...lineProps} />}
+      <LineSeriesRender {...lineProps} />
+      <ErrorBars {...errorBarsProps} />
       <ScatterSeries
         {...otherProps}
-        hidden={!displayMarker || props.hidden}
+        hidden={!displayMarker}
         groupId={id}
+        displayErrorBars={false}
```

Comment:
we usually do not specify boolean props unless their value needs to be `true`
```suggestion
        displayErrorBars={false}
```